data_collection/gazette/spiders/ap_tartarugalzinho.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class ApTartarugalzinhoSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "1600709"
+    name = "ap_tartarugalzinho"
+    start_urls = ["http://ap.portaldatransparencia.com.br/prefeitura/tartarugalzinho/"]
```

Comment:
The last edition dates from 1/Janeiro/2021. Could you take a look at the current gazettes' website?

Reply:
They've moved to this URL http://tartarugalzinho.ap.gov.br/diario_oficial, which also seems pretty straightforward to scrap. I wonder what's the procedure for when a city needs 2 or more spiders depending on date. Just add some "if" statements to point to one or another scraping function?

Reply:
I think so (at least it is what is done in other spiders). But feel free to open an issue, so this can be tackled later.